doc/basic_configuration.rst

```diff
@@ -55,17 +59,34 @@ In *all* build files::
     emails:
     - your_email@example.com
 
+You need to have a `local smtp service configured <https://github.com/ros-infrastructure/buildfarm_deployment#setup-master-for-email-delivery>`_ for email notifications.
+Note that even when you remove these global email notification settings
+some jobs will still send notification emails to package specific email addresses.
```

Comment:
Maybe note that this is configuration dependent. It can be disabled - by default it will send notification emails though.

Reply:
Thanks. Added a note.